src/cloud-cli/userdb.ts

```diff
@@ -148,6 +160,52 @@ export function migrate(): number {
     return 1;
   }
 
+  // Create DBOS system DB and tables.
+  // TODO: replace this with knex to manage schema.
+  const userPoolConfig: PoolConfig = {
+    host: configFile.database.hostname,
+    port: configFile.database.port,
+    user: configFile.database.username,
+    password: configFile.database.password,
+    connectionTimeoutMillis: configFile.database.connectionTimeoutMillis || 3000,
+    database: configFile.database.user_database,
+  };
+
+  try {
+    if (configFile.database.ssl_ca) {
```

Comment:
It doesn't feel good to have all this code in migrate, especially as it's already somewhere else.  Can we have a "create system database" function?

Reply:
Updated the code.